platform/core.windows/src/org/netbeans/core/windows/view/EditorView.java

```diff
@@ -257,6 +250,64 @@ public void dropActionChanged(DropTargetDragEvent dtde) {
                 setOpaque( false);
         }
 
+        private JComponent initBackgroundComponent() {
+            JComponent imageComponent = null;
+            String imageSource = Constants.SWITCH_IMAGE_SOURCE; // NOI18N
+            if (imageSource == null) {
+                FileObject config = FileUtil.getConfigFile("Windows2/Background");
+                if (config != null) {
+                    Object attr = config.getAttribute("backgroundImage");
+                    if (attr instanceof String) {
+                        imageSource = (String) attr;
+                    }
+                }
+            }
+            if (imageSource != null) {
+                Image image = ImageUtilities.loadImage(imageSource);
+                if (image != null) {
+                    JLabel label = new JLabel(ImageUtilities.image2Icon(image));
+                    label.setMinimumSize(new Dimension(0, 0)); // XXX To be able shrink the area.
+                    imageComponent = label;
+                } else {
+                    Logger.getLogger(EditorView.class.getName()).log(Level.WARNING, null,
+                            new java.lang.NullPointerException("Image not found at "
```

Comment:
nitpick: `java.lang.` is redundant here

Reply:
Well, not my code, this was moved from elsewhere in the file.  But have removed the redundant text in latest update.